File: src/main/java/de/smart/organizr/dto/NotePostDto.java
package de.smart.organizr.dto;

import lombok.AllArgsConstructor;
import lombok.Getter;

@AllArgsConstructor
@Getter
public class NotePostDto {
	private final int parentId;
	private final String title;
	private final int authorId;
	private final String description;
	private final int numberOfPages;
}

File: api_go/controllers/dto/NotePostDto.go
package dto

type NotePostDto struct {
	AuthorId      string `json:"authorId"      validate:"required"`
	ArrangerId    string `json:"arrangerId,omitempty"`
	Description   string `json:"description"   validate:"required"`
	NumberOfPages int    `json:"numberOfPages" validate:"required"`
	ParentId      string `json:"parentId"      validate:"required"`
	Name          string `json:"name"          validate:"required"`
	PdfContent    string `json:"pdfContent"`
}
